### PR TITLE
Add T2

### LINKED ISSUE #221: Remove dekode/wp-reset when T2 is production ready
Resets is a part of T2 so the composer package is no longer needed when T2 i released.

### PR DESCRIPTION
**Pull request by walbo**
Closes #221 

Also added package sorting

### CODE REVIEW

**Comment by olethomas** [comment]
Heads-up: Site-health is included in T2. It is possible to turn this on/off using the extension admin panel.

**Review by olethomas — APPROVED** [review]

File: packages/themes/dekode-theme/src/style.css [hunk 1]
```diff
@@ -32,18 +27,19 @@ img {
 	}
 }
 
-.entry-header {
+.site-container {
```
Comment (stian-overasen) [inline packages/themes/dekode-theme/src/style.css:30]: Could this helper class be moved to T2 @walbo? We usually need this on all projects, especially in header and footer.

**Comment by olethomas** [comment]
A second pair of eyes on this would be highly appreciated. @stian-overasen , @zpaceinvader , @Clorith

**Review by stian-overasen — APPROVED** [review]

**Comment by olethomas** [comment]
💥 🚀

**Review by olethomas — APPROVED** [review]

**Pull request merged**